client/src/components/molecules/AccountStatus/Indicator.module.scss

```diff
@@ -4,7 +4,7 @@
     display: inline-block;
     position: relative;
     cursor: help;
-    padding: .5rem;
+    padding: 0.5rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)